src/admin-dhis2/components/AdminDhis2Page/AdminDhis2Page.jsx

```diff
@@ -13,10 +13,82 @@
  * http://www.gnu.org/licenses.  For additional information contact info@OpenLMIS.org. 
  */
 
-import React from 'react';
+import React, { useMemo, useEffect, useState } from 'react';
 
-const AdminDhis2Page = () => (
-    <div></div>
-);
+import getService from '../../../react-components/utils/angular-utils';
+import Table from '../../../react-components/table/table';
+import TrashButton from '../../../react-components/buttons/trash-button';
+import ResponsiveButton from '../../../react-components/buttons/responsive-button';
 
-export default AdminDhis2Page;
\ No newline at end of file
+const AdminDhis2Page = () => {
+    const [serversParams, setServersParams] = useState([]);
+
+    const serversData = useMemo(
+        () => {
+            return getService('adminDhis2');
+        },
+        []
+    );
+
+    useEffect(
+        () => {
+            serversData.get()
+                .then((fetchedServer) => {
+                    const { content } = fetchedServer
+
+                    const serversParams = content.map((server) => ({
+                        serverId: server.id,
+                        Name: server.name,
```

Comment:
Can you rename this one:
Name -> name?
Please do this to have consistent naming conventions.

Reply:
sure will rename the whole conventions